association_backend/association/auth/entities.py

```diff
@@ -0,0 +1,69 @@
+from __future__ import annotations
```

Comment:
I don't think your service should have all of this 🤔

Reply:
i need this when i use this snippet:

```
class Something:
   def give_me_something(self): -> Something. # this throws NameError: name 'Something' is not defined
        pass
```

We can avoid it in python 3.10

I removed it where it was not necessary

Reply:
sorry I meant that the association shouldn't have an `auth` namespace, not the future annotations, you can keep it everywhere it's not an issue :)